### DISCUSSION TITLE
Icons for Properties

### DISCUSSION THREAD

**Discussion by Mountain1094** [FlorianWoelki/obsidian-icon-folder#278] [Ideas]
I would like to use the plugin to set icons for the file properties (introduced with Obsidian 1.4.0.

There is a beta plugin that is already doing that: https://github.com/Quorafind/Obsidian-File-Property-Enhancer

But I would like to see it in Obsidian Iconize.

**Comment by FlorianWoelki** [comment]
This is now added in v2.7.0!
